### DISCUSSION TITLE
Skip single digit on analog gauge

### DISCUSSION THREAD

**Discussion by SciLor** [jomjol/AI-on-the-edge-device#2592] [Ideas]
I have a watermeter where the second analog meter cannot be read due to the glass.
Is there anything In could do to skip one gauge?

![Screenshot_20230827-211944](https://github.com/jomjol/AI-on-the-edge-device/assets/1385855/bc9357b5-3eb7-43a7-a40b-17ae08125934)

**Comment by caco3** [comment]
The point still looks readable, so you could try to train your own model for it
See the documentation for details.

**Reply by SciLor** [reply]
I don't think so as gauge two is not readable from 5 - 7.

**Reply by caco3** [reply]
Then your only option is to skip this and all lower digits.
With the [ExtendedResolution](https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#PostProcessing-NUMBER.ExtendedResolution) you still should get an accuracy of << 100l!
